frontend/public/locales/fr/applications.json

```diff
@@ -1,25 +1,83 @@
 {
   "index": {
-    "page-title": "(FR) Applications",
+    "page-title": "Previous applications",
```

Comment:
Just noticed that the English content is in `/fr/applications.json` while the French content is in `/en/application.json`.